imager_images/admin.py

```diff
@@ -1,12 +1,10 @@
 from django.contrib import admin
-from models import Album, Photo
-from forms import *
 from django.db import transaction
 from django.contrib.admin.options import csrf_protect_m
 from sorl.thumbnail import get_thumbnail
-from imager import settings
-import os
 from imager_images.filters import PhotoSizeFilter
+from imager_images.forms import *
```

Comment:
Importing splat is generally bad practice. It works in this case, but it's better not to use it to keep your namespace controlled.